modules/dmrpp_module/SuperChunk.h

```diff
@@ -81,6 +81,7 @@ class SuperChunk {
     std::shared_ptr<http::url> get_data_url() { return d_data_url; }
     virtual unsigned long long get_size() const { return d_size; }
     virtual unsigned long long get_offset() const { return d_offset; }
+    virtual size_t get_chunks_size() const { return d_chunks.size(); }
```

Comment:
I'd prefer to name this something like `get_chunk_count()` or `get_num_chunks()`, but `get_chunks_size` is used elsewhere for this same concept, and having function name parity seemed useful.